src/pages/docs/game/chat.mdx

```diff
@@ -0,0 +1,20 @@
+---
+layout: ../../../layouts/docs.astro
+---
+
+
+# __Chat__ 
+[@veritanuda](https://keybase.io/veritanuda)
+
+In game chat is now possible by simply clicking the `Chat` button in the bottom left of the viewport window.
```

Comment:
I would refrain from using time references like "is now possible" as this will no longer be the case in let's say 6 months. What you can do is mention the release that added it as that is a clear reference point. (which in this case would be the upcoming 2024.3 release)

Reply:
Yeah..I was going to put a version number there.. but I realise I had no idea when it was added officially as I started playing around with it right away. I can nix that.